src/platforms/hosted/remote_jtagtap.c

```diff
@@ -126,26 +126,19 @@ static void jtagtap_tdi_tdo_seq(
 			chunk = 64;
 		}
 		ticks -= chunk;
-		uint8_t di[8];
-		memset(di, 0, 8);
+		uint64_t di = 0;
 		int bytes = (chunk + 7) >> 3;
-		if (DI) {
```

Comment:
Why the removal of the DI check? This looks like it introduces nullptr issues with its removal

Reply:
Colateral damage.

Reply:
DI check put back in place.